test/test_jacobians.jl

```diff
@@ -123,4 +123,14 @@ for fsym in ForwardDiff.auto_defined_unary_funcs
             throw(err)
         end
     end
-end
\ No newline at end of file
+
+    # Test jacobian of functions involving `.-`, `.+`, etc. #
+    #-------------------------------------------------------#
+    a = ones(N)
+    ops = [:-, :+]
+    for op in ops
+        @eval fn(x) = ($op)(a, x)
```

Comment:
It doesn't seem like this macro-fu saves much space and it's pretty hard to read, could you rephrase these tests more simply?